dot/node.go

```diff
@@ -241,7 +241,7 @@ func newNode(cfg *Config,
 	}
 
 	logger.Patch(log.SetLevel(cfg.Global.LogLvl))
-
+	logger.Debugf("node config %+v\n", cfg)
```

Comment:
nit do we still need this?

Reply:
I thought this would be a useful debugging message (config used to start node), so I left it since it's logging to debug.  Thoughts?

Reply:
Not bad to be honest.
Ideally we would have a nicely formatted cfg logged even at the info level, but that's for another day 😉